frontend/src/main/webapp/WEB-INF/config.sample.properties

```diff
@@ -95,3 +95,28 @@ module.path = {full path to "odcs" (home) dir of the project}/target
 #header.color1 = #5f5247
 #header.color2 = #5f5247
 #header.color3 = #5f5247
+
+#URL which needs to be called if the application wants to logout the CAS session
+logout.url=http://192.168.6.73:18080/unifiedviews/cas_logout
+
+#CAS server URL
+cas.server.host=https://192.168.1.8:8443/cas2
+
+#Callback URL for spring to authenticate the user.
+cas.security.check.url=http://192.168.6.73:18080/unifiedviews/login/cas
+
+#CAS login URL
+cas.login.url=https://192.168.1.8:8443/cas2/login
+
+#CAS logout URL
+cas.logout.url=https://192.168.1.8:8443/cas2/logout?service=http%3A%2F%2F192.168.7.42%3A18080%2Funifiedviews%2F
+
+#type of ownership user|organization
+ownership.type=organization
```

Comment:
if you mean ownership.type look for usage of cz.cuni.mff.xrg.odcs.commons.app.conf.ConfigProperty.OWNERSHIP_TYPE

if the value is "organization" entity is visible by all members of an organization, from which entity creator is comming. if value is "user" visibility should remain unchanged